src/index.js

```diff
@@ -3,6 +3,10 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import store from './redux/configureStore';
+import { getBooks } from './redux/books/books';
+
+store.dispatch(getBooks());
 
```

Comment:
- [ ] This request should be made in the `componentDidMount` lifecycle method or in a hook lifecycle function. Notice that you can use hooks like `useEffect` only in functional components, therefore you can not run that request in this file. It is common to make requests into components that need the requested data.

Reply:
Well solved